frontend/admin/resources/js/components/skillFamily/SkillFamilyTable.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useMemo } from "react";
+import { useIntl } from "react-intl";
+import { getLocale } from "@common/helpers/localize";
+import commonMessages from "@common/messages/commonMessages";
+import { useLocation } from "@common/helpers/router";
+import { notEmpty } from "@common/helpers/util";
+import { FromArray } from "@common/types/utilityTypes";
+import {
+  AllSkillFamiliesQuery,
+  useAllSkillFamiliesQuery,
+} from "../../api/generated";
+import Table, { ColumnsOf } from "../Table";
+import DashboardContentContainer from "../DashboardContentContainer";
+import { tableEditButtonAccessor } from "../TableEditButton";
+
+type Data = NonNullable<FromArray<AllSkillFamiliesQuery["skillFamilies"]>>;
+
+export const SkillFamilyTable: React.FC<
+  AllSkillFamiliesQuery & { editUrlRoot: string }
+> = ({ skillFamilies, editUrlRoot }) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const columns = useMemo<ColumnsOf<Data>>(
+    () => [
+      {
+        Header: intl.formatMessage({
+          defaultMessage: "ID",
+          description: "Title displayed on the Skill Family table ID column.",
+        }),
+        accessor: "id",
+      },
+      {
+        Header: intl.formatMessage({
+          defaultMessage: "Name",
+          description:
+            "Title displayed for the Skill Family table Name column.",
+        }),
+        accessor: (sf) => sf.name?.[locale],
+      },
+      {
+        Header: intl.formatMessage({
+          defaultMessage: "Description",
+          description:
+            "Title displayed for the Skill Family table Description column.",
+        }),
+        accessor: (sf) => sf.description?.[locale],
+      },
+      {
+        Header: intl.formatMessage({
+          defaultMessage: "Category",
+          description:
+            "Title displayed for the Skill Family table Category column.",
+        }),
+        accessor: "category",
```

Comment:
I think this should be localized as well.  It's an enum so check the localizedstrings files for examples.